.ci/docker/ubuntu/Dockerfile

```diff
@@ -8,6 +8,7 @@ ENV DEBIAN_FRONTEND noninteractive
 
 ARG CLANG_VERSION
 
+
```

Comment:
wont this still result in a cache hit in the docker build cache? #130591 should force a rebuild since docker will hash the file when it is copied into the image.